From: Richard Hughes <richard@hughsie.com>
Date: Wed, 25 Jan 2012 16:32:18 +0000 (+0000)
Subject: Fix the ColorHug patch to not time out with firmware >= 1.1.1
X-Git-Url: http://pkgs.fedoraproject.org/gitweb/?p=argyllcms.git;a=commitdiff_plain;h=50e52fa4219f358580c3a07d8157c01c61c6d7a9

Fix the ColorHug patch to not time out with firmware >= 1.1.1
---

diff --git a/0001-Add-an-experimental-ColorHug-sensor-driver.patch b/0001-Add-an-experimental-ColorHug-sensor-driver.patch
index 2edd6e8..38a7990 100644
--- a/0001-Add-an-experimental-ColorHug-sensor-driver.patch
+++ b/0001-Add-an-experimental-ColorHug-sensor-driver.patch
@@ -301,7 +301,7 @@ index 0000000..90f1b7e
 +}
 +
 +/* Converts a packed float to a double */
-+static double packed_float_to_double (uint32_t pf)
++static double packed_float_to_double (int32_t pf)
 +{
 +	return (double) pf / (double) 0x10000;
 +}
@@ -349,7 +349,7 @@ index 0000000..90f1b7e
 +	ev = colorhug_command(p, ch_take_reading,
 +						  ibuf, sizeof (ibuf),
 +						  (unsigned char *) obuf, sizeof (obuf),
-+						  1.0);
++						  5.0);
 +	if (ev != inst_ok)
 +		return ev;
 +
